### ISSUE TITLE
Make Syscollector logging messages more user-friendly

### ISSUE THREAD

**Issue by chemamartinez** [wazuh/wazuh#3384]
|Wazuh version|Component|Install type|Install method|Platform|
|---|---|---|---|---|
| v3.9.1 | Agentd/Control module | Agent | All | All |

Since v3.9.0, the keep-alive file sent by agents includes the primary IP asked from the Agent daemon to a new module which uses the network inventory functions to collect it. The goal is to agents registered without IP can be identified by this new one. 

This module is working on Linux, Windows and Mac OS X agents and it is enabled by default. The problem here is there is no way of disabling it right now, so potential errors when collecting the network information cannot be avoided. So, errors like the following one appear every time the keep-alive file is created (every 10 seconds by default).

```
WARNING: Unable to read MAC address for interface "eth0:1" from "/sys/class/net/eth0:1/address": No such file or directory (2)
```

### Proposal solutions

- [x] Make Syscollector error messages more user-friendly. Example:
https://github.com/wazuh/wazuh/blob/b3dc8c2fd5f0c9c554bc24537d23a84307d5f3da/src/wazuh_modules/syscollector/syscollector_linux.c#L1766

**Comment by Skeptor** [comment]
I'm working in the branch https://github.com/wazuh/wazuh/tree/issue-3384
The commit https://github.com/wazuh/wazuh/commit/113afc613b3da23ac24cdffafdb5e7fa33a6cb8d add to the configuration of the agent the option to disable the control module to avoid infinite errors in case the agent can't extract the network information

**Comment by Skeptor** [comment]
When an agent had reported his IP before, the IP in the `global.db` will continue to be the reported IP even the configuration has changed. We have to define if this is our desired behaviour.

The commit https://github.com/wazuh/wazuh/commit/4de574dbc8a90c6297d34e60aafd38f7a5f3306d change some error logs to be more user-friendly and less cryptic.

Memory Reports
------------------------
Valgrind result when the option is configured in the `ossec.conf` as `yes`
![image](https://user-images.githubusercontent.com/9034923/58340404-19625600-7e4c-11e9-82f7-269956b33bb7.png)


Valgrind result when the option is configured in the `ossec.conf` as `no`
![image](https://user-images.githubusercontent.com/9034923/58340543-67775980-7e4c-11e9-8b2e-7bfc367bba42.png)

**Comment by Skeptor** [comment]
The commit  https://github.com/wazuh/wazuh/commit/89a3fc9269218b1fdcda2aad8da0f7741693ffec adds the variable `report_ip` to the function `getClientConfig` to be returned by the API.

![image](https://user-images.githubusercontent.com/9034923/58425831-2aed6d00-809b-11e9-8b06-43495d151269.png)

**Comment by chemamartinez** [comment]
Merged #3397
